library/core/src/main/java/com/google/android/exoplayer2/text/cea/Cea608Decoder.java

```diff
@@ -284,6 +328,24 @@ protected void decode(SubtitleInputBuffer inputBuffer) {
       // If we've reached this point then there is data to process; flag that work has been done.
       captionDataProcessed = true;
 
+      if ((ccDataHeader & CC_VALID_FLAG) != CC_VALID_FLAG) {
+        // The screen needs to be cleared when the closed captions are turned off
```

Comment:
Is this defined in a specification somewhere? The information I've managed to find suggests the pair should be ignored / treated as padding. Which is what the implementation did prior to this change.

Reply:
* I agree, the standard is quite ambiguous and miss describing requirements.
* Seemingly some caption providers simply set this flag to "not valid" and do not put any other data or command into the stream. This is the case for live (over the air) TV, 608 is not used for current digital on-demand formats. Missing this would cause, for example, the last caption of the previous movie kept continuously on for an entire new movie without captions. As this new movie does not have any valid data, there is no other signs to clear the screen for. 
* The [Sarnoff](https://www.sri.com/engage/products-solutions/sarnoff-bitstreams-decoder-tests) CEA608-708 verification documents clearly say that as soon as the flag is set to "not valid", the screen must be empty. Unfortunately, these test streams are not free.

Reply:
Ok, although if this turns out to break anyone we will likely revert it to be in line with what the specification says.